### ISSUE TITLE
Access to 'Reader View' within app

### ISSUE THREAD

**Issue by Gandalf-the-Blue** [JGeek00/my-linkding#18]
Hi,
It would be great to also have access to the Reader view and other views as well, potentially within the app that can provide a good reading experience without going into the browser.
Thanks!

**Comment by JGeek00** [comment]
What do you mean with reader mode? Is that a Linkding feature?

**Comment by JGeek00** [comment]
I'm checking it and I don't have that option 🤨
<img width="1840" alt="image" src="https://github.com/JGeek00/linkdy/assets/47545344/419d163a-afff-4965-9339-ac8fd489157a">

**Comment by JGeek00** [comment]
Yes, I'm using latest tag. I will work on this when it's available for everyone.

**Comment by privx-de** [comment]
I miss this feature too. `latest-plus` is available and stable for everyone. The different tags only have to do with the required resources. It would be great if the feature was also in Linkdy.
